samples/extensions/dynamic_rendering/dynamic_rendering.cpp

```diff
@@ -80,6 +83,7 @@ bool DynamicRendering::prepare(vkb::Platform &platform)
 		create_render_pass_non_dynamic();
 	}
 	create_pipeline();
+	transition_swapchain();
```

Comment:
This modifies the swap-chain images prior to them being acquired, which is not VU IIRC. What might be as well, if this is done only to get the images into `VK_IMAGE_LAYOUT_PRESENT_SRC_KHR` layout prior to frame rendering, is to simply transition UNDEFINED -> BLAH every frame. There are no downsides to that approach as far as I know.